src/tests/ftest/datamover/negative.py

```diff
@@ -242,18 +242,21 @@ def test_dm_bad_params_dcp(self):
             "POSIX", "/fake/fake/fake",
             expected_rc=1)
 
-    @skipForTicket("DAOS-6871")
     def test_dm_negative_space_dcp(self):
         """Jira ID: DAOS-5515
         Test Description:
             DAOS-5515: destination pool does not have enough space.
             DAOS-6387: posix filesystem does not have enough space.
         :avocado: tags=all,full_regression
-        :avocado: tags=datamover,dcp,dfuse
+        :avocado: tags=datamover,dcp
         :avocado: tags=dm_negative,dm_negative_space_dcp
         """
         self.set_tool("DCP")
 
+        # mount small tmpfs filesystem on posix path, using size required sudo
+        dst_posix_path = self.new_posix_test_path(mount_dir=True)
+        self.execute_cmd("sudo mount -t tmpfs none '{}' -o size=128M".format(dst_posix_path))
```

Comment:
@daltonbohning that is fine I can move it, but what other tests need to mount? I am not aware of any. I would prefer to avoid it in a test honestly.

Reply:
@daltonbohning moved it!

Reply:
> @daltonbohning that is fine I can move it, but what other tests need to mount? I am not aware of any. I would prefer to avoid it in a test honestly.

I was thinking of the large tests that need to be updated, and also in case we have future tests that need similar logic. In the test framework in general, there has been some effort to centralize and consolidate logic for various tasks. So this falls into that general mindset.

Reply:
@daltonbohning yeah I am working on updating the large tests and the /mnt/share space is already setup so it doesn't require mounting, but makes sense to centralize it.